Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM python:3.11-alpine
+FROM python:3.11
```

Comment:
Not able to keep it on alpine?

Reply:
mm I can't remember the context here, but I think scheduling breaks when not on alpine?